client/lomas_client/client.py

```diff
@@ -693,25 +198,21 @@ def get_previous_queries(self) -> Optional[List[dict]]:
                         pass
                     case DPLibraries.SMARTNOISE_SYNTH:
                         return_model = query["client_input"]["return_model"]
-                        res = query["response"]["query_response"]
+                        res = query["response"]["result"]
                         if return_model:
-                            query["response"]["query_response"] = pickle.loads(
+                            query["response"]["result"] = pickle.loads(
                                 base64.b64decode(res)
                             )
                         else:
-                            query["response"]["query_response"] = pd.DataFrame(res)
+                            query["response"]["result"] = pd.DataFrame(res)
                     case DPLibraries.OPENDP:
                         opdp_query = make_load_json(
                             query["client_input"]["opendp_json"]
                         )
                         query["client_input"]["opendp_json"] = opdp_query
                     case DPLibraries.DIFFPRIVLIB:
-                        model = base64.b64decode(
-                            query["response"]["query_response"]["model"]
-                        )
-                        query["response"]["query_response"]["model"] = pickle.loads(
-                            model
-                        )
+                        model = base64.b64decode(query["response"]["result"]["model"])
```

Comment:
Not necessarily in this PR but to have some pydantic checks everywhere, we could add a pydantic model for query responses. Do you plan to do it ? Otherwise we can open a ticket.

Reply:
I think we should indeed to it here. I'll open a ticket